modules/core/nuxeo-core-storage-sql/nuxeo-core-storage-sql/src/main/java/org/nuxeo/ecm/core/storage/sql/jdbc/JDBCBackend.java

```diff
@@ -126,8 +126,9 @@ public Model initialize(RepositoryImpl repository) {
         }
         if (log.isDebugEnabled()) {
             FulltextDescriptor fulltextDescriptor = repositoryDescriptor.getFulltextDescriptor();
-            log.debug(String.format("Database ready, fulltext: disabled=%b searchDisabled=%b.",
-                    fulltextDescriptor.getFulltextDisabled(), fulltextDescriptor.getFulltextSearchDisabled()));
+            log.debug(String.format("Database ready, fulltext: disabled=%b storedInBlob=%b searchDisabled=%b.",
+                    fulltextDescriptor.getFulltextDisabled(), fulltextDescriptor.getFulltextStoredInBlob(),
+                    fulltextDescriptor.getFulltextSearchDisabled()));
```

Comment:
Worth updating the class to log4j 2, nop?

Reply:
Sure but later :)
Note that this debug log is only seen once at startup so no performance-critical.